nncf/common/logging/track_progress.py

```diff
@@ -20,20 +20,22 @@
 from rich.progress import Task
 from rich.progress import TaskProgressColumn
 from rich.progress import TextColumn
-from rich.progress import TimeElapsedColumn
-from rich.progress import TimeRemainingColumn
 from rich.style import StyleType
 from rich.text import Text
 
+INTEL_BLUE_COLOR = (0, 113, 197)
```

Comment:
`INTEL_BLUE_COLOR_STYLE = "rgb(0,113,197)"`
To avoid `text_style = f"rgb({INTEL_BLUE_COLOR[0]},{INTEL_BLUE_COLOR[1]},{INTEL_BLUE_COLOR[2]})"`

Reply:
![image](https://github.com/openvinotoolkit/nncf/assets/48012821/7f2e300c-7407-4301-aa2d-bba14b14915c)
On intel.com background of logo is `0, 104, 181` 😉

Reply:
Done